sdk/src/azure/core/az_mqtt5_rpc_server.c

```diff
@@ -10,41 +10,23 @@
 
 AZ_NODISCARD az_mqtt5_rpc_server_options az_mqtt5_rpc_server_options_default()
 {
-  return (az_mqtt5_rpc_server_options){
-    .subscribe_timeout_in_seconds = AZ_MQTT5_RPC_DEFAULT_TIMEOUT_SECONDS,
-  };
+  return (az_mqtt5_rpc_server_options){ .subscribe_timeout_in_seconds
+                                        = AZ_MQTT5_RPC_DEFAULT_TIMEOUT_SECONDS,
+                                        .subscription_topic_format = AZ_SPAN_EMPTY };
 }
 
 AZ_NODISCARD az_result
 az_rpc_server_get_subscription_topic(az_mqtt5_rpc_server* client, az_span out_subscription_topic)
```

Comment:
Functions must be named according to the object.
Since the object is `az_mqtt5_rpc_server` the function name must be `az_mqtt5_rpc_server_get_subscription_topic`. Same with all other codec public APIs.